pkgs/data/themes/materia-theme-transparent/default.nix

```diff
@@ -0,0 +1,24 @@
+{
+  lib,
+  fetchFromGitHub,
+  materia-theme,
+}:
+materia-theme.overrideAttrs (oldAttrs: rec {
+  pname = "materia-theme-transparent";
+  version = "20210322";
+
+  src = fetchFromGitHub {
+    owner = "ckissane";
+    repo = pname;
+    rev = "c5d95bb";
```

Comment:
```suggestion
    rev = "c5d95bbddd59a717bfc4976737af429a89ba74e0";
```

Reply:
What exactly does the appended hash do? I'm still a beginner to Nix.

Reply:
Alright :)